exec/internal/compile/backend_amd64.go

```diff
@@ -344,6 +356,28 @@ func (b *AMD64Backend) paramsForMemoryOp(op byte) (size uint, inst obj.As) {
 	panic("unreachable")
 }
 
+func (b *AMD64Backend) emitFusedConstStore(builder *asm.Builder, regs *dirtyRegs, code []byte, nextInst InstructionMetadata, ci, nextCI currentInstruction, imm uint64) error {
```

Comment:
this new function doesn't seem to be tested either.

Reply:
Added test.